template/python3-armhf/template.yml

```diff
@@ -0,0 +1,2 @@
+language: python3-arhmf
```

Comment:
Small typo here - should be `armhf`. Please can you amend the commit and push again?